### ISSUE TITLE
Training object recognition model with and without resuming

### ISSUE THREAD

**Issue by nargeshn** [keras-team/keras#11262]
I am trying to train a model for object classification in Keras. Initially, I trained my model for 400 epochs and then to train for 600 epochs I used two following ways:

1) I considered the weight for the trained model with 400 epochs as initial weight and trained for 200 more epochs (image 1).

2) I started the training from the beginning for 600 epochs (image 2).

I expected to see the same behavior in both of the aforementioned cases but it wasn't the case. You can find the accuracy and loss for both scenarios in the following images.
Also, the learning rate does not change during the process.

<img width="541" alt="screenshot 2018-09-29 23 31 13" src="https://user-images.githubusercontent.com/30293166/46259518-71f17700-c4a8-11e8-8bac-8a04193568be.png">
<img width="538" alt="screenshot 2018-09-29 23 33 02" src="https://user-images.githubusercontent.com/30293166/46259519-728a0d80-c4a8-11e8-832b-93d22694f247.png">
